invenio_records_resources/services/custom_fields/base.py

```diff
@@ -28,10 +41,13 @@ def mapping(self):
     @property
     @abstractmethod
     def field(self):
-        """Marshmallow schema for vocabulary custom fields."""
+        """Marshmallow schema for vocabulary custom fields.
+
+        Must call _field_kwargs() to get the kwargs for the field.
+        """
         pass
 
     @property
     def ui_field(self):
         """Marshmallow UI schema for vocabulary custom fields."""
```

Comment:
same here, although this property needs to be checked if it's a field or a schema